application/src/main/java/org/thingsboard/server/actors/tenant/TenantActor.java

```diff
@@ -271,9 +275,24 @@ private TbActorRef getOrCreateDeviceActor(DeviceId deviceId) {
                 () -> new DeviceActorCreator(systemContext, tenantId, deviceId));
     }
 
-    private void onToEdgeSessionMsg(EdgeEventUpdateMsg msg) {
-        log.trace("[{}] onToEdgeSessionMsg [{}]", msg.getTenantId(), msg);
-        systemContext.getEdgeRpcService().onEdgeEvent(tenantId, msg.getEdgeId());
+    private void onToEdgeSessionMsg(TbActorMsg msg) {
+        switch (msg.getMsgType()) {
```

Comment:
I don't like this approach. Since this is all related to EdgeRpcService it should be handled there and not in the TenantActor.